tools/roll_release.sh

```diff
@@ -94,9 +94,8 @@ fi
 
 echo ""
 echo "Setting the signing key for signing the release. It is up to you to change it back to your own afterwards."
-git config user.signingkey E18AE35E86BB194F
-git config user.email "contact@stellar-group.org"
-git config user.name "STE||AR Group"
+git -c user.signingkey E18AE35E86BB194F -c user.email contact@stellar-group.org \
+    -c user.name "STE||AR Group"
```

Comment:
@gonidelis the `-c` flags belong on the `git tag` call. From `man git`:
```
       -c <name>=<value>
           Pass a configuration parameter to the command. The value given will
           override values from configuration files. The <name> is expected in
           the same format as listed by git config (subkeys separated by
           dots).
```

Reply:
oh... my bad. that would be three `git tag` commands command one after the other, no?

Reply:
oh my bad...But I am still confused. What do you mean "belong"? I thought `git -c` is equivalent to `git config`, no?
So the proper code should be:

```
git -c user.signingkey E18AE35E86BB194F 
git -c user.email contact@stellar-group.org 
git -c user.name "STE||AR Group"
```
?

Reply:
> @msimberg Or do I completely remove the `git config/-c` command and just embed `-c <name>=<value>` in the that line.

Yep!